runtime/src/main/scala/akka/grpc/internal/ServerReflectionImpl.scala

```diff
@@ -52,6 +55,9 @@ final class ServerReflectionImpl private (fileDescriptors: Map[String, FileDescr
       ServerReflectionResponse(req.host, Some(req), response)
     })
   }
+
+  private def getProtoBytes(fileDescriptor: FileDescriptor): ByteString =
+    protoBytesLocalCache.getOrElseUpdate(fileDescriptor.getName, fileDescriptor.toProto.toByteString)
```

Comment:
I think this isn't thread-safe, is it? Might be worth picking a concurrent implementation, perhaps even just good old `ConcurrentHashMap`

Reply:
Indeed it's not thread safe this way, I'll make the change !

Reply:
Done !